src/test/run-make-fulldeps/emit-stack-sizes/Makefile

```diff
@@ -0,0 +1,25 @@
+-include ../tools.mk
+
+ifdef IS_WINDOWS
+# Do nothing on MSVC.
+all:
+	exit 0
+else
+# check that the .stack_sizes section is generated
+# this test requires LLVM >= 6.0.0
+vers = $(shell llvm-ar -version)
```

Comment:
This is not the right way to check the version of LLVM linked into *rust*. It will check for the system llvm instead. You can do `rustc -Cllvm-args=-version`, instead.

I wanted to suggest to use a codegen test that checks the assembly. Alas that is currently not possible, so a run-make test will have to do.

Reply:
You probably want `rustc -vV` instead, that has a line that's e.g. `LLVM version: 8.0` on the current nightly.

Reply:
> You probably want rustc -vV instead

Good call. I had forgotten that the LLVM version was in that output.